### PR TITLE
[Examples][BugFix] Update runtime used in examples

### PR DESCRIPTION
**Pull request by zhengfeiwang**
# Description

- Update runtime used in examples; currently use a CI owned by SP.
- Update the README for chat-with-pdf example to avoid different behavior from different shell.

# All Promptflow Contribution checklist:
- [x] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: examples/tutorials/run-management/cloud-run-management.ipynb [hunk 1]
```diff
@@ -376,7 +376,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.17"
+   "version": "3.10.12"
```
Comment (crazygao) [inline examples/tutorials/run-management/cloud-run-management.ipynb:379]: don't change this currently.

File: examples/tutorials/get-started/quickstart-azure.ipynb [hunk 2]
```diff
@@ -426,7 +426,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.17"
+   "version": "3.10.12"
```
Comment (crazygao) [inline examples/tutorials/get-started/quickstart-azure.ipynb:429]: revert this line.

File: examples/flows/chat/chat-with-pdf/README.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ pf flow test --flow . --inputs question="What is the name of the new language re
 run_name="web_classification_"$(openssl rand -hex 12)
 
 # run with multiline data, --name is optional
-pf run create --flow . --data ./data/bert-paper-qna-3-line.jsonl --stream --column-mapping question='${data.question}' pdf_url='${data.pdf_url}' chat_history='${data.chat_history}' config='{ \"EMBEDDING_MODEL_DEPLOYMENT_NAME\": \"text-embedding-ada-002\", \"CHAT_MODEL_DEPLOYMENT_NAME\": \"gpt-35-turbo\", \"PROMPT_TOKEN_LIMIT\": \"2000\", \"MAX_COMPLETION_TOKENS\": \"256\", \"VERBOSE\": \"True\", \"CHUNK_SIZE\": \"256\", \"CHUNK_OVERLAP\": \"32\" }' --name $run_name
```
Comment (crazygao) [inline examples/flows/chat/chat-with-pdf/README.md:47]: save the original format since this is demonstration of manual input of multiple lines of data.

Comment (crazygao) [reply examples/flows/chat/chat-with-pdf/README.md:47]: or save one and change the other one.

File: examples/tutorials/get-started/quickstart-azure.ipynb (hunk 2 above)
Comment (zhengfeiwang) [reply examples/tutorials/get-started/quickstart-azure.ipynb]: reverted

File: examples/tutorials/run-management/cloud-run-management.ipynb (hunk 1 above)
Comment (zhengfeiwang) [reply examples/tutorials/run-management/cloud-run-management.ipynb]: reverted

**Review by crazygao — APPROVED** [review]
lgtm

**Review by brynn-code — APPROVED** [review]

**Pull request merged**